Modules/Utils.cs

```diff
@@ -122,14 +122,13 @@ public static (string, Color) GetRoleTextHideAndSeek(RoleTypes oRole, CustomRole
         public static bool HasTasks(GameData.PlayerInfo p, bool ForRecompute = true)
         {
             //Tasksがnullの場合があるのでその場合タスク無しとする
-            if (p.Tasks == null) return false;
-            if (p.Role == null) return false;
+            if (p.Tasks == null || p.Role == null)
+                return false;
 
             var hasTasks = true;
-            if (p.Disconnected) hasTasks = false;
-            if (p.Role.IsImpostor)
+            if (p.Disconnected || p.Role.IsImpostor)
                 hasTasks = false; //タスクはCustomRoleを元に判定する
-            if (Options.CurrentGameMode == CustomGameMode.HideAndSeek)
+            else if (Options.CurrentGameMode == CustomGameMode.HideAndSeek)
```

Comment:
Impostor saves are counted in HideAndSeek

Reply:
Fixed with 7b78cb36f81204ab7423765d3e9c128809b5d12d

Reply:
No, don’t use “else”

Reply:
Revert back? 
```
...
            var hasTasks = true;
            if (p.Disconnected) hasTasks = false;
            if (p.Role.IsImpostor)
                hasTasks = false; //タスクはCustomRoleを元に判定する
            if (Options.CurrentGameMode == CustomGameMode.HideAndSeek)
            {
                if (p.IsDead) hasTasks = false;
                var hasRole = Main.AllPlayerCustomRoles.TryGetValue(p.PlayerId, out var role);
                if (hasRole)
                {
                    if (role is CustomRoles.HASFox or CustomRoles.HASTroll) hasTasks = false;
                }
            }
            else
...
```

Suggestions?

Reply:
Yes, please revert.

Reply:
Reference
746411adfa7db7a19da61af3eb5b347ac1d8b2d1